jobs/cjd_sample/src/main/java/com/google/samples/LocationSearchSample.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples;
+
+import com.google.api.services.jobs.v2.JobService;
+import com.google.api.services.jobs.v2.model.JobQuery;
+import com.google.api.services.jobs.v2.model.LocationFilter;
+import com.google.api.services.jobs.v2.model.RequestMetadata;
+import com.google.api.services.jobs.v2.model.SearchJobsRequest;
+import com.google.api.services.jobs.v2.model.SearchJobsResponse;
+import java.io.IOException;
+import java.util.Arrays;
+
+/** Location Search */
+public final class LocationSearchSample {
+
+  private static JobService jobService = JobServiceUtils.getJobService();
+
+  // [START basic_location_search]
+  /** Basic location Search */
+  public static void basicLocationSearch() throws IOException {
+    // Make sure to set the requestMetadata the same as the associated search request
+    RequestMetadata requestMetadata =
+        new RequestMetadata()
+            .setUserId("HashedUserId") // Make sure to hash the userID
+            .setSessionId("HashedSessionID") // Make sure to hash the sessionID
+            .setDomain("www.google.com"); // Domain of the website where the search is conducted
+    LocationFilter locationFilter =
+        new LocationFilter().setName("Mountain View, CA").setDistanceInMiles(0.5);
```

Comment:
Being able to specify basic things like the name and distance here as parameters would make this sample much more interactive and engaging for users.

Reply:
There are 5 test cases. Do you mean we should let the customer to specify the method, location and distance?